client/src/__mocks__/gatsby.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable no-unused-vars */
-const React = require('react');
+import { LinkProps } from '@reach/router';
```

Comment:
```suggestion
import { GatsbyLinkProps } from 'gatsby';
```
As I understand it, we're trying to replace gatsby's link, not reach router's.